notification/mem_storage.go

```diff
@@ -0,0 +1,33 @@
+package notification
+
+import (
+	"math/rand"
+	"strconv"
+)
+
+type MemStorage struct {
+	events   []Event
+	latestID uint64
+}
+
+func NewMemStorage() *MemStorage {
+	return &MemStorage{latestID: rand.Uint64()}
+}
+
+func (m MemStorage) add(event Event) error {
+	m.events = append(m.events, event)
```

Comment:
This should have a limited capacity to avoid memory exhaustion.

Reply:
This is just a dummy code and shall be improved